plugins/atlas/privatekeypair.go

```diff
@@ -0,0 +1,52 @@
+package atlas
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func PrivateKeyPair() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.PrivateKeyPair,
```

Comment:
Question: Glancing at the docs, isn't this credential what Mongo calls the "API Key"? Which consists of a private and public key: https://www.mongodb.com/docs/atlas/configure-api-access/#std-label-atlas-admin-api-access

Reply:
<img width="829" alt="image" src="https://user-images.githubusercontent.com/7430639/234537012-a3463764-1d40-4189-87e1-dff333448cf4.png">

Reply:
I agree with your suggestion. I've updated the credential names/ filenames accordingly. Please let me know if you have any other feedback.

Reply:
And would it make sense to use [this page](https://www.mongodb.com/docs/atlas/configure-api-access/) as the API Key's `DocsURL`?